### ISSUE TITLE
Paper Key Re-Keying Message Loop

### ISSUE THREAD

**Issue by rminnick** [keybase/client#3937]
On my new computer, I'm getting the paper-key re-keying message in a loop - it pops up on boot and every few hours while on the computer.

I've attached the screenshot of the specific notice.
![screen shot 2016-08-17 at 3 51 01 pm](https://cloud.githubusercontent.com/assets/14830996/17750891/b537451c-6492-11e6-963d-7f913b8b1b7b.png)

my log id: 63c038447f53e6ee7173861c

**Comment by maxtaco** [comment]
Thanks for the feedback.  We've fixed most of these problem in our master branch, hopefully we can get a release out to you soon!

**Comment by maxtaco** [comment]
BTW, for now you can safely ignore the problem.

**Comment by maxtaco** [comment]
Hmm, actually this might be a rekey bug if you've entered your paper key.

Cc: @jzila and @strib

**Comment by rminnick** [comment]
I can confirm that I registered my computer with my paper key.  I had re-installed the OS and used the paper key to both add the computer and to re-key my FS folders when prompted.

**Comment by jzila** [comment]
If you added the computer with the paper key, you shouldn't have been prompted to re-enter it as long as the new client stayed on after provisioning.

**Comment by jzila** [comment]
I'm looking at your logs now

**Comment by jzila** [comment]
@rminnick from your logs, it looks you provisioned the new device with a paper key, but as soon as it was done KBFS immediately rekeyed your folders except for your private folder.

When did you create /keybase/private/ryanminnick, and from which device?

**Comment by rminnick** [comment]
I created it from the same machine, but an old OS install.  I had re-installed my OS yesterday afternoon.  I'm assuming that the confusion was that the devices were switched out simultaneously instead of in sequence.  The notice asking for my key seemed to read to that effect - something along the lines of "if you don't turn on the other device or enter your paper key, you'll lose information forever."

**Comment by jzila** [comment]
Yeah that sounds right. Ok so the only remaining issue is why /keybase/private/ryanminnick isn't getting rekeyed. Have you tried accessing it?

**Comment by jzila** [comment]
Sorry let me rephrase that. The critical issue is to determine why that folder didn't get rekeyed. If you keep getting the popup, that's an issue too but I think might be unrelated.

**Comment by rminnick** [comment]
It's been re-keyed, or at least I can access it.  The issue now remains as to why it feels a need to display that paper key notice periodically.

**Comment by jzila** [comment]
kk, can you do another log send?

**Comment by rminnick** [comment]
Sure - here's the new log ID:  e12a0d58f01b64433c1d011c

**Comment by jzila** [comment]
OK I've verified that all your folders are keyed correctly, and KBFS isn't setting the flag that asks for a paper key.

@rminnick: I'm opening an internal ticket for this issue, and we'll notify you here when we fix it.

**Comment by rminnick** [comment]
Thanks John!

**Comment by jzila** [comment]
@rminnick looks like this was fallout from your account reset; we're working on fixing it.

**Comment by jzila** [comment]
OK you should be good now. LMK if it's still happening.
